app/ai/voice/agents/breeze_buddy/managers/__init__.py

```
@@ -0,0 +1,367 @@
+                    await _attempt_fallback_call(
+                        session,
+                        locked_lead,
+                        number_to_use,
+                        config,
+                        template,
+                        use_template_flow,
+                    )
+
+                await release_lock_on_lead_by_id(locked_lead.id)
+
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)